### DISCUSSION TITLE
Startup Issue

### DISCUSSION THREAD

**Discussion by sunixter1** [quarto-dev/quarto-nvim#69] [Q&A] [answered]
I'm getting this error at startup

Failed to run " config' for nvim-lspconfig
/Users/sramos/.config/nvim/lua/plugins/quarto. lua:303: attempt to concatenate local 'resour ce path' (a nil value)
# stacktrace:
- ~/ .config/nvim/lua/plugins/quarto. lua: 303
- /quarto-nvim/lua/quarto/init. lua: 3
- /quarto-nvim/plugin/quarto. lua: 16
- vim/ editor. lua:0 _in_
**cmd**
- ~/.config/nvim/lua/config/lazy. lua: 17
- ~/.config/nvim/init. lua:2
in_
**config**

**Comment by jmbuhr** [comment, marked as answer]
Do you have quarto installed?

**Reply by sunixter1** [reply]
Yes, thru the git clone instructions

**Reply by sunixter1** [reply]
sorry my bad... no, I missed it and thought it was all done thru git clone..  I'm good now.  Thank you for your reply
